frontend/src/store/wallet.ts

```diff
@@ -329,8 +329,8 @@ export default function useWalletStore() {
       const [_hasEnsKeys, _hasCnsKeys] = _isAccountSetup
```

Comment:
*arbitrary comment location*

Was the intent here to remove the warning for legacy setup? I didn't think so, but that seems to be the effect:

Production:

![image](https://user-images.githubusercontent.com/1479802/231213643-2c93784b-280a-4056-975c-e9638c82a71c.png)

Deploy Preview:

![image](https://user-images.githubusercontent.com/1479802/231213944-7b4bb041-b6bb-424e-b7fe-38a4d8528720.png)

Reply:
Hmm, interesting. What's the status of the address you're logged in with? It has legacy keys setup on an ENS resolver but never registered in the StealthKeyRegistry?

Reply:
Ok, investigating this. On main:
- _isAccountSetup: false
- _hasEnsKeys: true
- _hasCnsKeys: false

On this branch:
- isAccountSetup: false
- _hasEnsKeys: false
- _hasCnsKeys: false

Seems the ENS lookup throws on this branch with the below error, not yet sure why:

```
v2.0.1 WARNING - Error: processing response error (body="{\"jsonrpc\":\"2.0\",\"id\":48,\"error\":{\"code\":-32000,\"message\":\"the method has been deprecated: eth_accounts\"}}\n", error={"code":-32000}, requestBody="{\"method\":\"eth_accounts\",\"params\":[],\"id\":48,\"jsonrpc\":\"2.0\"}", requestMethod="POST", url="<REDACTED_BY_ME>", code=SERVER_ERROR, version=web/5.7.1)
    at Logger.makeError (index.js:224:1)
    at Logger.throwError (index.js:233:1)
    at eval (index.js:261:1)
    at Generator.next (<anonymous>)
    at fulfilled (index.js:5:1)
```

Reply:
Ok should be fixed per bca969c, details about the fix in https://github.com/ScopeLift/umbra-protocol/pull/488/files#r1163363489,

Reply:
Fixed in 1c3177b